index.html

```diff
@@ -1,14 +1,188 @@
 <!DOCTYPE html>
 <html class="page" lang="EN">
+
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,400,700" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=PT+Serif:700i" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.0.0/normalize.css">
   <link rel="stylesheet" href="dist/style.css">
 </head>
+
 <body>
+  <header class="header">
+    <div class="container">
+      <div class="top-bottom-line">
+        <ul class="menu header__menu">
+          <li class="menu__item header__menu-item"><a href="#" class="link menu__link">log in</a></li>
+          <li class="menu__item header__menu-item"><a href="#" class="link menu__link">sign up</a></li>
+          <li class="menu__item header__menu-item">
+            <a href="#" class="link menu__link">
+          <img src="img/menu.png" alt="menu">
+        </a>
+          </li>
+        </ul>
+        <img class="header__logo" src="img/logo.png" alt="logo">
+      </div>
+      <h2 class="heading heading_2 header__subheading">Find your Favorite</h2>
+      <h1 class="heading heading_1 header__heading">recipes</h1>
+      <button class="button header__button">get started</button>
+      <p class="header__text">Or send us your own recipes and <a class="header__link" href="#">get awarded!</a></p>
+    </div>
+  </header>
+
+  <main>
```

Comment:
it is better not to use just tags -- each tag should be either component or element -- so if you need -- you can match it in css

Reply:
fixed